charts/consul/test/unit/terminating-gateways-deployment.bats

```diff
@@ -1236,6 +1236,24 @@ key2: value2' \
   [ "${actual}" = "bar" ]
 }
 
+@test "terminatingGateways/Deployment: vault namespace annotations can be set when secretsBackend.vault.vaultNamespace is set and .global.secretsBackend.vault.agentAnnotations is not set." {
```

Comment:
I think this test is currently failing: https://github.com/hashicorp/consul-k8s-workflows/actions/runs/6149328162/job/16692536295#step:6:2216

Reply:
This was running fine on my local. Had to make some syntactical changes to make it pass on CI.